AsyncDisplayKit/Layout/ASLayoutable.h

```diff
@@ -9,16 +9,23 @@
  */
 
 #import <AsyncDisplayKit/ASDimension.h>
+#import <AsyncDisplayKit/ASRelativeSize.h>
 #import <AsyncDisplayKit/ASStackLayoutDefines.h>
+#import <AsyncDisplayKit/ASStackLayoutable.h>
+#import <AsyncDisplayKit/ASBaselineLayoutable.h>
+#import <AsyncDisplayKit/ASStaticLayoutable.h>
+
+#import <AsyncDisplayKit/ASLayoutablePrivate.h>
 
 @class ASLayout;
+@class ASLayoutSpec;
 
 /** 
  * The ASLayoutable protocol declares a method for measuring the layout of an object. A class must implement the method
  * so that instances of that class can be used to build layout trees. The protocol also provides information 
  * about how an object should be laid out within an ASStackLayoutSpec.
```

Comment:
Update this comment, probably referencing ASLayoutOptions as the container of metadata required for layout, giving Stack as an example but referencing the possibility of custom extensions ("see ASLayoutOptions")